### DISCUSSION TITLE
AGU Abstract

### DISCUSSION THREAD

**Discussion by dcherian** [xarray-contrib/cf-xarray#459] [General]
This AGU session seems like a great fit for a cf-xarray abstract if anyone here was interested (cc @kthyng )

Anyone is welcome to present about cf-xarray!

-------

Title: IN017 - CF and NetCDF: 30 Years of Wide Open Science
Link: https://agu.confex.com/agu/fm23/prelim.cgi/Session/192249
Deadline for Abstract Submission: 2 Aug 2023
Conveners: Ethan Davis, Kevin O'Brien, Charlie Zender, David Hassel
Description:

The CF (Climate and Forecast) Conventions are a community-developed standard for describing Earth system science data in the netCDF data format. The CF Conventions can encode information that describes the coordinate systems, data structure, and geophysical meaning and units of each variable, and how the data were collected. It is widely used by weather and climate scientists and remote-sensing researchers and is gaining traction in new communities, such as biogeochemistry and operational weather prediction. It has a mature ecosystem of FOSS and commercial software tools that can explore, analyze, and visualize data that is encoded using the CF Conventions.

This session will focus on efforts to extend the existing CF Conventions; recent advances in the CF and netCDF software ecosystem; the experience of projects in domains and features new to CF and netCDF (e.g., new standard names, compression, aggregation, Zarr); optimization of data formats; and engagement with the community.

**Comment by dcherian** [comment]
I put it in after the deadline. Let's see what happens 🤷🏾‍♂️

**Reply by aulemahal** [reply]
Thanks for submitting the abstract. I haven't done much for `cf-xarray` in the last months, so thanks for including me in the authors!

If this gets accepted and you have options to edit the metadata, I think it would be better to put "Ouranos" as my institution. Haha, I graduated almost 4 years ago now ;)

**Comment by kthyng** [comment]
Sorry I've been traveling since SciPy and have been dropping lots of balls. Don't plan to make it to fall AGU, but might be able to swing ocean sciences.
